src/gwt/src/org/rstudio/studio/client/RStudioGinModule.java

```diff
@@ -23,18 +24,14 @@
 import org.rstudio.core.client.VirtualConsole;
 import org.rstudio.core.client.VirtualConsole.PreferencesImpl;
 import org.rstudio.core.client.VirtualConsoleFactory;
+import org.rstudio.studio.client.application.*;
```

Comment:
We generally don't use wildcard imports, please revert back to individual ones.

See: "Avoid using wildcard (org.path.to.package.*) imports" on this page: https://github.com/rstudio/rstudio/wiki/Coding-Standards-(java,-cpp)